icons/webmin.svg

```diff
@@ -1 +1 @@
 6.7972 3.0058c.102 1.2269.209 2.4728.313 3.7287-1.08.455-2.2178.934-3.4277 1.4459-1.255.536-2.1229 1.6089-2.4569 2.8848-.34 1.2999-.135 2.8077.723 4.1426.763 1.195 1.8558 2.0059 3.2547 2.4449-.08 1.0919.03 2.0798.34 2.9547.138.389.316.755.533 1.098.798 1.2688 1.9778 2.0288 3.1827 2.2348a3.7827 3.7827 0 003.2938-.998c.9799-.9449 1.9068-1.8378 2.7898-2.6837 1.0469.63 2.0558 1.2349 3.0377 1.8268 1.604.957 3.2428-.084 3.6298-2.1148.162-.847.09-1.7919-.21-2.8618.57-.351.9249-.901 1.0659-1.6709.305-1.6339-.503-3.7817-1.8269-4.8796-.702-.582-1.4269-1.177-2.1638-1.7829l-.005-.023c-.186-1.0499-.367-2.0798-.548-3.0897-.306-1.7239-1.7988-3.5378-3.3737-4.0397-.722-.23-1.3929-.158-2.0349.225-.7789-.824-1.5928-1.38-2.4798-1.664A3.6817 3.6817 0 009.3 0zm1.019 2.4788c.12-.003.25.012.381.048.4669.125.9118.49 1.1548.932.198.368.29.6699.37 1.2978l.404 3.1428c.013.099.027.195.038.293.115.889.644 1.5279 1.3299 1.6189.659.087 1.0099-.43.8899-1.26a12.249 12.249 0 00-.038-.278l-.442-3.0047c-.063-.425-.084-.75-.05-.96.052-.4649.422-.7069.896-.5789.479.131.9269.572 1.1339 1.097.106.266.154.4729.236.9639.157.9409.312 1.8898.472 2.8677.013.087.028.174.039.267.019.09.045.169.068.25l.002.014.001.002.023.084v-.002c0-.004 0-.007-.002-.01l.003.007v.003c.099.374.3439.746.7059 1.03.065.052.139.103.205.1599.7299.573 1.4418 1.14 2.1408 1.6859.421.33.573.484.74.717.2689.3799.3819.9338.2869 1.3588-.087.393-.341.627-.658.59-.26-.031-.48-.137-.9239-.465-.734-.55-1.4839-1.112-2.2598-1.6839-.068-.048-.144-.1-.213-.152-.646-.476-1.2079-.385-1.4239.265-.222.666.09 1.449.754 1.9139.072.05.153.107.223.158.8099.567 1.6048 1.1199 2.3798 1.6598.329.229.573.444.735.638.3569.409.5269 1.08.4039 1.6419-.132.593-.514.9019-.978.796-.235-.05-.411-.144-.8319-.417-.817-.53-1.6569-1.079-2.5228-1.637-.08-.05-.16-.104-.237-.16-.273-.166-.537-.242-.768-.233h-.002c-.2599-.017-.5309.078-.7918.286-.072.06-.146.117-.213.18-.809.666-1.647 1.375-2.5348 2.1219-.48.4-.696.557-1.017.67-.645.246-1.3579-.03-1.8038-.739-.421-.692-.404-1.5889.032-2.1538.185-.252.498-.54.9299-.846.94-.672 1.8299-1.3049 2.6758-1.9118.077-.052.152-.104.228-.159.641-.455.764-1.26.312-2.0109-.44-.7499-1.17-.9759-1.8599-.559a5.3066 5.3066 0 00-.235.146c-.895.549-1.8338 1.128-2.8238 1.739-.638.384-.9759.5089-1.4519.5309-.59.03-1.2049-.332-1.5638-.904-.407-.6359-.421-1.4268-.03-1.9178.22-.288.455-.463 1.1299-.809 1.0429-.5399 2.0448-1.0478 2.9947-1.5248.082-.05.163-.084.247-.125.058-.029.112-.063.159-.097.425-.188.632-.657.566-1.2929-.012-.107-.02-.21-.037-.312a375.2283 375.2283 0 00-.351-3.3157c-.07-.66-.063-.892.021-1.154.1-.3129.39-.4929.752-.5029zM7.1091 6.7355l.002.026a.022.022 0 01-.003-.005v-.018l.001-.002zm.004.043l.006.087c-.003-.026-.005-.051-.01-.08 0-.003.003-.005.004-.008zm.01.136l.003.025h-.002v-.015c0-.003.002-.005 0-.008v-.001zm.005.04v.012c-.001-.003-.003-.005-.003-.008.002 0 .002-.002.003-.004zm10.0992 2.3218l.002.012c0 .002 0 .004.003.007l-.006-.018zm.005.019c.005.015.008.034.013.05l-.01-.044a.011.011 0 01-.003-.007z"/></svg>
```

Comment:
Looks like there's a few differences between PR (red) and source (black) - could you look into this, and see if you can get it down to 3 decimals please?

![image](https://user-images.githubusercontent.com/1830867/103170924-f7a72d00-483f-11eb-8c49-bcfa9124cca8.png)

Reply:
@adamrusted I've now rebuilt it from source (and changed the source as well to point to the specific SVG file).

It should match pretty well now. If you compare them, make sure that you use the same "version" as I do. I've removed all the "3D effects" and shadows.

Note: A precision of 3 didn't pass the linter. I had to use a precision of 4 instead.